src/model/queries/ticket.js

```diff
@@ -0,0 +1,30 @@
+const dbConnection = require('../dbConnection');
+
+const getLatestTicket = () => new Promise((resolve, reject) => {
```

Comment:
ASALA why are creating new promise while it's already available !!!

Reply:
I let them do it like this last time so they can learn how to write a promise from scratch. But as Salwa said, no need to define a new promise. dbConnection.query(sql) itself is a promise.